src/applications/personalization/dashboard/components/benefit-payments-and-debts/DismissibleDebts.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CTALink from '../CTALink';
+import recordEvent from '~/platform/monitoring/record-event';
+import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
+import '../../sass/user-profile.scss';
+
+export const Debts = ({ debts, hasError }) => {
+  const debtsCount = debts?.length || 0;
+  const [dismissed, setDismissed] = React.useState(false);

```

Comment:
This is temporary. This will be hooked up to the VANotify APIs to dismiss the alert